jenkins-integration/beaker/install/foss/30_install_dev_repos.rb

```diff
@@ -42,21 +42,17 @@ def get_latest_master_version(version)
 end
 
 def get_latest_agent_version
-  url = "https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent%20suite%20pipelines/job/platform_puppet-agent_intn-van-promote_suite-daily-promotion-master/lastSuccessfulBuild/api/json"
+  url = "http://builds.puppetlabs.lan/passing-agent-SHAs/api/v1/json/report-master"
```

Comment:
soo, aren't we phasing out the use of `*.puppetlabs.lan`?

Reply:
Probably, that's just the url that was given to me but it looks like builds.delivery.puppetlabs.net works too so I'll update it.

Reply:
Updated.